spec/models/user_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
```

Comment:
This file doesnt do anything, so we shouldn't commit it. When/if we have specs to add to Users, we can add it then.